Tests/fontBuilder/fontBuilder_test.py

```diff
@@ -244,3 +244,34 @@ def test_setupNameTable_no_windows():
 
     assert all(n for n in fb.font["name"].names if n.platformID == 1)
     assert not any(n for n in fb.font["name"].names if n.platformID == 3)
+
+
+def test_unicodeVariationSequences(tmpdir):
+    familyName = "UVSTestFont"
+    styleName = "Regular"
+    nameStrings = dict(familyName=familyName, styleName=styleName)
+    nameStrings['psName'] = familyName + "-" + styleName
+    glyphOrder = [".notdef", "space", "zero", "zero.slash"]
+    cmap = {ord(" "): "space", ord("0"): "zero"}
+    uvs = [
+        (0x0030, 0xFE00, "zero.slash"),
+        (0x0030, 0xFE01, "zero"),  # not an official sequence, just testing
```

Comment:
maybe you could also add a test case where the third item is None

Reply:
would be nice if the test data reflected some realistic unicode variation sequence, e.g. the zero.slashed that @moyogo used elsewhere. I think it was `U+0030 U+FE00`

Reply:
oops.. never mind (i must be drunk)

Reply:
It's a mix of realistic and unrealistic :)